### PR TITLE
r/ssm_maintenance_window: Fix ssm maintenance window

### PR DESCRIPTION
**Pull request by atsushi-ishibashi**
`enabled` isn't set correctly via create cycle.

### CODE REVIEW

File: aws/resource_aws_cloudtrail.go [hunk 1]
```diff
@@ -99,7 +99,7 @@ func resourceAwsCloudTrailCreate(d *schema.ResourceData, meta interface{}) error
 	if v, ok := d.GetOk("cloud_watch_logs_role_arn"); ok {
 		input.CloudWatchLogsRoleArn = aws.String(v.(string))
 	}
-	if v, ok := d.GetOk("include_global_service_events"); ok {
+	if v, ok := d.GetOkExists("include_global_service_events"); ok {
```
Comment (Ninir) [inline aws/resource_aws_cloudtrail.go]: Not sure this change needs to be part of this work. Can you confirm @atsushi-ishibashi?

Comment (atsushi-ishibashi) [reply aws/resource_aws_cloudtrail.go]: @Ninir Sorry, I checked out from #2817 
I'll remove this commit!

**Comment by atsushi-ishibashi** [comment]
@Ninir Ok👍

**Comment by radeksimko** [comment]
Just reran the mentioned acceptance test, all good, so I'll merge this as I plan to look into one bug in this resource (and avoid conflicts :) ).

@Ninir I hope you don't mind.

**Comment by radeksimko** [comment]
Thanks @atsushi-ishibashi for the PR!

**Comment by bflad** [comment]
This has been released in [terraform-provider-aws version 1.7.0](https://github.com/terraform-providers/terraform-provider-aws/blob/master/CHANGELOG.md#170-january-12-2018). Please see the [Terraform documentation on provider versioning](https://www.terraform.io/docs/configuration/providers.html#provider-versions) or reach out if you need any assistance upgrading.

**Pull request merged**